gophish-main.tf

```diff
@@ -0,0 +1,112 @@
+provider "aws" {
+  region     = "eu-central-1"
```

Comment:
We might consider avoiding specifying a default region here. I could see people surprised that the instance is spun up in `eu-central-1`. Instead, it [looks like](https://www.terraform.io/docs/providers/aws/) the `AWS_DEFAULT_REGION` environment variable is respected by Terraform, so perhaps we could just document that that variable is required?

What are your thoughts on this?

Reply:
Yes, you are right. Didn't think of this when I wrote the template. I'll change it like described [here](https://www.terraform.io/docs/providers/aws/#environment-variables)